pages/blog/index.js

```diff
@@ -95,7 +100,7 @@ export default function BlogIndexPage() {
             />
             {showClearFilters && (
               <span className="text-sm leading-10">
-                <Link href="/blog" passHref><a> Clear filters </a></Link>
+                <button onClick={clearFilters} className="">Clear filters</button>
```

Comment:
I think that we should here button like this:

![image](https://user-images.githubusercontent.com/20404945/173110325-07f87a94-3109-4dc5-a559-be37ab7e1721.png)

You can copy code from that line

https://github.com/asyncapi/website/blob/master/pages/index.js#L280

However, please remember about mobile view :) Let me know if you will have some problems.